components/locale/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "icu-locale"
+description = "API for managing Unicode Langauge and Locale Identifiers"
+version = "0.0.1"
+authors = ["The ICU4X Project Developers"]
+edition = "2018"
+readme = "README.md"
+repository = "https://github.com/unicode-org/omnicu"
+license = "MIT/Apache-2.0"
+categories = ["internationalization"]
+include = [
+    "src/**/*",
```

Comment:
@Manishearth should we remove src/ part of each package?

Reply:
I'm in favor of keeping `src` directory alongside with `tests`, `benches`, `docs` and `examples` per component.

Reply:
I'll leave you and Manish to duke it out :)